sdk/ai/ai-projects/src/evaluations/customModels.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { RequestParameters } from "@azure-rest/core-client";
+import { Evaluation } from "../generated/src/models.js";
+import { ListQueryParam, UpdateBodyParam } from "../generated/src/parameters.js";
+
+export interface CreateEvaluationBodyParam {
+  /** Evaluation to run. */
+  body: Evaluation;
+}
+
+export type CreateEvaluationParameters = CreateEvaluationBodyParam & RequestParameters;
+export type GetEvaluationParameters = RequestParameters;
+
+export type ListEvaluationParameters = ListQueryParam & RequestParameters;
+
+export type GetEvaluationScheduleParameters = RequestParameters;
+
+export type CreateOrReplaceEvaluationScheduleParameters = RequestParameters;
+
+export type ListEvaluationScheduleParameters = ListQueryParam & RequestParameters;
+
+export type DisableEvaluationScheduleParameters = RequestParameters;
+
+export type UpdateEvaluationParameters = UpdateBodyParam & RequestParameters;
```

Comment:
Curious why these need to be defined ?

Reply:
I was following a similar pattern to agents and connections. Each wrapper function has a parameter option type that matches the name of the function, even if there is overlap between the types.

Reply:
These are not necessary and can directly use the models in from the generated models. As these are not exported to be consumed by evaluation operations.

Reply:
Updated this to use only generated models